.gitignore

```diff
@@ -1,4 +1,7 @@
-.vscode
+*.json
+!.vscode/*.json
+!.devcontainer/*.json
+
```

Comment:
I would add the `.env` just in case as well.

Reply:
Fair point, added